test/filter.js

```diff
@@ -1,5 +1,3 @@
-global.window = global;

```

Comment:
why this?

Reply:
@joao-conde Noticed that it was an error that was left in the code from Gabriel's time, that was never needed.

Reply:
The comment made at the time https://github.com/ripe-tech/ripe-commons-js/pull/33#discussion_r845975089